src/main/kotlin/net/irisshaders/lilybot/utils/DatabaseHelper.kt

```diff
@@ -156,6 +156,48 @@ object DatabaseHelper {
 		return collection.find(TagsData::guildId eq inputGuildId).toList()
 	}
 
+	/**
+	 * Using the provided [inputThreadId] the owner's ID or null is returned from the database.
+	 *
+	 * @param inputThreadId The ID of the thread you wish to find the owner for
+	 *
+	 * @return null or the thread owner's ID
+	 * @author tempest15
+	 */
+	suspend fun getThreadOwner(inputThreadId: Snowflake): Snowflake? {
+		val collection = database.getCollection<ThreadData>()
+		val selectedThread = collection.findOne(ThreadData::threadId eq inputThreadId) ?: return null
+		return selectedThread.ownerId
```

Comment:
Is there not another way to return this? maybe return `selectedThread` instead, for consistency

Reply:
Given that `ThreadData` only includes the thread's id and the owner's id there's no point. Returning `selectedThread` would be giving redundant information.

Reply:
Good point, well made